### ISSUE TITLE
Issue with apps package filters

### ISSUE THREAD

**Issue by Doff3n** [Adobe-Marketing-Cloud/aem-project-archetype#42]
Any reason for the /apps/install package being excluded in the apps filters?

```
<?xml version="1.0" encoding="UTF-8"?>
<workspaceFilter version="1.0">
    <filter root="/apps/${appsFolderName}">
        <exclude pattern="/apps/${appsFolderName}/install" />
    </filter>
    <filter root="/apps/sling" />
    <filter root="/etc/designs/${appsFolderName}"/>
</workspaceFilter>
```

It causes issues when exporting built packages at a later point. The core.jar file is then not included in builds, which means you have to install that manually on different servers?(We do not only use maven)

**Comment by rombert** [comment]
I _think_ it's done this way to prevent conflicts with `vlt`, but maybe @eglistefan recalls, as we created the project initially.

**Comment by eglistefan** [comment]
iirc the idea was to avoid having the core bundle installed twice - once directly via the core project itself and once via the ui.apps

**Comment by rombert** [comment]
This is now fixed in version 11 with #62
